float64 front_right
float64 front_left
float64 back_right
float64 back_left

